src/v2/Components/v2/ArtworkFilter/StickyContainer.tsx

```diff
@@ -0,0 +1,72 @@
+import styled, { css } from "styled-components"
+import { Box, Flex, color } from "@artsy/palette"
+import React, { useEffect, useRef, useState } from "react"
+import { NAV_BAR_HEIGHT } from "v2/Components/NavBar"
+
+const Container = styled(Flex).attrs({
+  top: NAV_BAR_HEIGHT,
+  py: 1,
+  mx: -2,
+  px: 2,
+  bg: "white100",
+  borderBottom: "1px solid",
+})<{ stuck?: boolean }>`
+  justify-content: space-between;
+  align-items: center;
+  position: sticky;
+  z-index: 1;
+
+  ${({ stuck }) =>
+    stuck
+      ? css`
+          border-bottom-color: ${color("black10")};
+        `
+      : css`
+          border-bottom-color: transparent;
+        `};
+`
+
+// This <div> is positioned such that when it leaves the top of
+// the browser the <Container> reaches it's `top` value and sticking.
```

Comment:
You can see how this works here: http://static.damonzucconi.com/_capture/n3ou4zmwl9tJ.gif (the red line is the sentinel)